packages/story-editor/src/components/library/panes/shopping/product.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { Text, THEME_CONSTANTS } from '@googleforcreators/design-system';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import ProductImage from './productImage';
+import ProductButton from './productButton';
+import ProductPrice from './productPrice';
+
+const StyledDescription = styled.div`
+  padding-left: 12px;
+`;
+
+const StyledTitle = styled(Text).attrs({
+  size: THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.MEDIUM,
```

Comment:
According to Figma the font size is supposed to be 14px, but `MEDIUM` is 16px.

```suggestion
  size: THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL,
```

Reply:
Update here
https://github.com/GoogleForCreators/web-stories-wp/blob/fix/11155/packages/story-editor/src/components/library/panes/shopping/productPrice.js#L28